languages/Developer/en.ini

```
@@ -0,0 +1,11 @@
+api_key = "API key"
+api_key_delete = "Delete"
+api_key_deletion_failed = "API key deletion failed"
+api_key_deletion_success = "API key was deleted successfully"
+api_key_generate = "Generate new"
+api_key_generation_failed = "API key generation failed"
+api_key_generation_success = "API key was generated successfully. Key will be displayed only once so save it: %%TOKEN%%"
+api_key_locked = "API key is locked. Please contact support if you have questions"
+settings = "Developer settings"
+show_developer_settings = "Show developer settings"
+verify_email_address = "Please verify your email-address to use developer settings"
```

We don't hyphenate "email address" in other strings; we should adjust this for consistency.

```suggestion
verify_email_address = "Please verify your email address to use developer settings"
```